### PR TITLE
[Issue #9] - Browser support table

### LINKED ISSUE #9: [FEATURE]: Browser Support Table on Home Page
## Summary

Add a Browser support table with the data provided https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl, check if it is possible to add column filter and sorting

Clear acknowledge the source of data, you see that around a lot 😄   

### Basic example

https://ant.design/components/table/#components-table-demo-head in the above table

### Motivation

This information is important enough to the users so don't make them have to jump between sites

### PR DESCRIPTION
**Pull request by JoaoJesus94**
Relates to #9 
Fixes #9 

## Screenshots (if visual changes)
image

### CODE REVIEW

File: src/components/BrowserSupportTable/index.js [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+              </div>
+            </th>
+            {headers.map(header => (
+              <Header key={header.name} name={header.name} />
+            ))}
+          </tr>
+        </thead>
+        <tbody>
+          {browserInfo.map(info => (
+            <Rows
```
Comment (RafaelFerreiraTVD) [inline src/components/BrowserSupportTable/index.js]: Hi @JoaoJesus94 

I think that we can pass only 1 prop to the Rows component, such as: **info**.

Then, on **Rows** Component, we can simplify a bit, using only the prop passed previously.

An example of Rows Component (not tested 😛)

```
const Rows = (info) => {
  return (
    <tr>
      <td>{info.browser}</td>
      Object.keys(info).filter(column => column !== 'browser').map(col => (
         <td className={getClassName(info[col])}>{info[col]}</td>
      ));
    </tr>
  );
};
```

When calling the Rows component, just use as you have but passing only the info prop.

Maybe something similar with the example below:

```
{browserInfo.map(info => <Rows info={info} />)}
```

This way, we are not hardcoding the way that we're building the rows, and if we need to add any new column, the only thing to do is simply add on `src/components/BrowserSupportTable/constants.js`

Comment (JoaoJesus94) [reply src/components/BrowserSupportTable/index.js]: I'll be using Table from antd on the future

File: src/components/BrowserSupportTable/styles.module.css [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+th.rotate > span > div > span {
+  padding: 5px 10px;
+}
+
+.tableWidth {
+  max-width: max-content;
+  margin: auto;
+}
+
+.myTest {
```
Comment (RafaelFerreiraTVD) [inline src/components/BrowserSupportTable/styles.module.css]: Are you using this css? If not, we can remove it 👍

Comment (JoaoJesus94) [reply src/components/BrowserSupportTable/styles.module.css]: Sure, thanks

**Pull request closed without merging**